Source/DafnyLanguageServer/Workspace/ProjectManager.cs

```diff
@@ -345,7 +346,7 @@ public class ProjectManager : IDisposable {
   }
 
   public void OpenDocument(Uri uri, bool triggerCompilation) {
-    Interlocked.Increment(ref openFileCount);
+    openFiles.TryAdd(uri, 1);
```

Comment:
What role does the `1` serve here? Could this be a set instead of a map?

Reply:
The only concurrent set I found in C# is a ConcurrentBag but it curiously does not have a method for removing a specific element.